namespaces/live-1.cloud-platform.service.justice.gov.uk/hmcts-complaints-formbuilder-adapter-production/resources/rds.tf

```diff
@@ -0,0 +1,30 @@
+module "hmcts-complaints-adapter-rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.5"
+
+  cluster_name               = "${var.cluster_name}"
+  cluster_state_bucket       = "${var.cluster_state_bucket}"
+  db_backup_retention_period = "${var.db_backup_retention_period_hmcts_complaints_adapter}"
+  application                = "hmcts-complaints-formbuilder-adapter"
+  environment-name           = "${var.environment-name}"
+  is-production              = "${var.is-production}"
+  infrastructure-support     = "${var.infrastructure-support}"
+  team_name                  = "${var.team_name}"
+  force_ssl                  = true
+  db_engine_version          = "11.4"
+
```

Comment:
Since this is a production RDS, let's make it bigger. Could you add : 
`db_instance_class = "db.m4.xlarge"`

Reply:
Hi @mtrblmoj 

Thanks for the suggestion.
This database won't be heavily used, will only have 1 table and the data won't be persisted for a very long time.  

What is the default size if not specified?

Reply:
👍